### PR TITLE
Provide default charm store URL

### LINKED ISSUE #45: API version should be set by the library instead of supplied by user
The comments and repo summary say to use v4 of the API, but, as reported in #43, that now fails with the current version of this library.

The library should have the supported API version in the code, instead of relying on it being provided by the user, so that a given release will always reference the API version that it works with.

### PR DESCRIPTION
**Pull request by johnsca**
Supports overriding the default URL for backwards compatibility.

Fixes #45

### CODE REVIEW

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/theblues-lib/159/
Test PASSed.

**Comment by pengale** [comment]
LGTM/+1 (disregard the earlier version of this comment complaining about formatting in the tests.)

File: theblues/utils.py [hunk 1]
```diff
@@ -15,6 +15,8 @@
 )
 
 
+API_URL = 'https://api.jujucharms.com/'
```
Comment (howbazaar) [inline theblues/utils.py]: I think it would be better and a more minimal change to just set the API_URL to 'https://api.jujucharms.com/v5'

**Comment by johnsca** [comment]
Simplified the diff based on review.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/theblues-lib/162/
Test PASSed.

**Review by howbazaar — APPROVED** [review]

**Comment by howbazaar** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/theblues-lib-merge

**Pull request merged**